### ISSUE TITLE
UI for advanced features

### ISSUE THREAD

**Issue by christiankaindl** [christiankaindl/trello-super-powers#13]
The ultimate goal for Trello Super Powers is that it can offer great integration with the users Trello account and make it easy to see notifications, search boards and manage boards and maybe even create cards or other tasks.

For this to be possible TSP will need a user interface in the future. Right now I am unsure whether that will be a Browser Action popup (a button in the toolbar), a completely separate page which can leverage the whole browser window, or a sidebar, which could yield interesting results because of its tall real estate and the fact that it is always there if you need it (popups close themselves and with separate pages you can't simultaneously do something different).

This is a big question that I will try to continuously look into, so that we can have an awesome 1.0 release. If someone has ideas in that regard, nothing is set in stone yet, so feel free to share it here.

**Comment by DAveShillito** [comment]
Personally I expect to see Firefox add-on options on menu from the Add-ons manager.
There seem to be two approaches people use for this, one is to open a whole new page with options.
i.e. "Tab Session Manger" (https://addons.mozilla.org/en-GB/firefox/addon/tab-session-manager)

The second is to embed these option in the add-on page 
i.e. "My Sessions" (https://addons.mozilla.org/en-GB/firefox/addon/my-sessions)

Of the two, personally I prefer the way "My Sessions" does it, but either work.

**Comment by christiankaindl** [comment]
Yes, embed into the add-on page is what TSP currently does. I also think it look better.

![screenshot from 2018-03-15 08 04 04](https://user-images.githubusercontent.com/15364860/37448639-7cf5c840-2827-11e8-9e06-9ccc4adc5aba.png)
